freqtrade/strategy/interface.py

```diff
@@ -285,6 +295,31 @@ def custom_stoploss(self, pair: str, trade: Trade, current_time: datetime, curre
         """
         return self.stoploss
 
+    def custom_sell(self, pair: str, trade: Trade, current_time: datetime, current_rate: float,
+                    current_profit: float, dataframe: DataFrame,
+                    **kwargs) -> Optional[Union[str, bool]]:
```

Comment:
I tried to be 100% mypy-compliant. Returning `None` is merely a convenience of not returning anything from overridden function as opposed to having to return `False` when no sell conditions were met. Saves one statement for the user. Not that it would break if `Optional` was dropped :]